backend/migrations/0003_create_users_table.up.sql

```diff
@@ -0,0 +1,11 @@
+-- create_users_table.up
+-- Prefer TEXT > VARCHAR: https://wiki.postgresql.org/wiki/Don%27t_Do_This#Don.27t_use_varchar.28n.29_by_defaultj
+CREATE TABLE IF NOT EXISTS users (
+    id BIGSERIAL NOT NULL,
+    username TEXT NOT NULL,
+    first_name TEXT NOT NULL,
+    last_name TEXT NOT NULL,
+    email TEXT NOT NULL,
+    password TEXT NOT NULL,
+    PRIMARY KEY (username, email)
```

Comment:
```suggestion
    id BIGSERIAL NOT NULL PRIMARY KEY,
    username TEXT NOT NULL UNIQUE,
    first_name TEXT NOT NULL,
    last_name TEXT NOT NULL,
    email VARCHAR(254) NOT NULL UNIQUE,
    password TEXT NOT NULL,
```

- `username, email` should not be the private key the private key should always be the `id` except for in some really weird instances.
  
- You should make `email` and `username` unique though.

- Use `varchar` since max email length is 254 characters anyway, no need to the extra length